plugins/webp-uploads/hooks.php

```diff
@@ -280,12 +280,16 @@ function webp_uploads_wp_get_missing_image_subsizes( $missing_sizes, $image_meta
  *
  * @since 1.0.0
  *
- * @param string $output_format The image editor default output format mapping.
- * @param string $filename      Path to the image.
- * @param string $mime_type     The source image mime type.
```

Comment:
Note that `webp_uploads_filter_image_editor_output_format()` is added as a filter for `image_editor_output_format`. This filter is defined as taking these params:

```
		 * @param string[] $output_format {
		 *     An array of mime type mappings. Maps a source mime type to a new
		 *     destination mime type. Default empty array.
		 *
		 *     @type string ...$0 The new mime type.
		 * }
		 * @param string $filename  Path to the image.
		 * @param string $mime_type The source image mime type.
```

So the use of `string` here for `$output_format` is incorrect. It should have been `array<string, string>` in the plugin, and in core `string[]` is not as specific as it could be (since it implies numeric keys).

Additionally, I tried using just `string` PHP types for `$filename` and `$mimetype`, but when running unit tests I got a failure. This is because the function that applies the filter is `\WP_Image_Editor::get_output_format()` and this function takes `null` as default values for `$filename` and `$mime_type`, which is what is passed when using a factory to create an image during testing. So core should actually be typing the filter:

```
		 * @param string|null $filename  Path to the image.
		 * @param string|null $mime_type The source image mime type.
```

These are accounted for with the new function signature for `webp_uploads_filter_image_editor_output_format()`.